cpp/simulations/abm_braunschweig.cpp

```diff
@@ -364,12 +366,13 @@ void create_world_from_data(mio::abm::World& world, const std::string& filename,
             }
             auto first_location_id = it_first_location_id->second.first;
             auto first_location    = locations.find(first_location_id)->second;
-            auto& person           = world.add_person(first_location, determine_age_group(age));
-            auto home              = locations.find(home_id)->second;
+            auto& person =
+                world.get_person(world.add_person(first_location, determine_age_group(age))); // TODO: is this safe?
+            auto home = locations.find(home_id)->second;
             person.set_assigned_location(home);
             person.set_assigned_location(hospital);
             person.set_assigned_location(icu);
-            persons.insert({person_id, person});
+            persons.insert({person_id, person}); // TODO: why? what?
```

Comment:
There are three, to be exact.

But granted, they are not especially well formulated. I think I was confused what this code is doing, in particular why we are adding a person twice (first to world, then using the insert, which is a reference to the world population). That is also why I ask if the line above is safe, because insertion here invalidates the Person& (which was not an issue with Person*)